### DISCUSSION TITLE
External page and Select::createOptionForm

### DISCUSSION THREAD

**Discussion by Ashk2a** [filamentphp/filament#7988] [Help] [answered]
Hello, I'm using a Select with `createOptionForm` in RegisterTenant and EditTenant.
It's working pretty well in edit page but not in register page, the modal doesn't appear.

I don't know if I need to add something to have the same behaviour in register page than edit page ?

Thanks

**Comment by leandrocfe** [comment, marked as answer]
RegisterTenant is a simple page (Filament\Pages\SimplePage)
I think it isn't working because simple page doesn't have `<x-filament-actions::modals />`

**Comment by Ashk2a** [comment]
Exactly !! Thanks a lot <3 

I override the register-tenant page, but I think it should really nice to add modals in simple page
